apps/arena/lib/arena/game_updater.ex

```diff
@@ -516,9 +515,16 @@ defmodule Arena.GameUpdater do
   defp maybe_receive_zone_damage(player, _elaptime, _zone_damage_interval, _zone_damage),
     do: player
 
+  # This method will decide what to do when a projectile has collided with something in the map
+  # - If collided with something with the same owner skip that collision
+  # - If collided with external wall or obstacle explode projectile
+  # - If collided with another player do the projectile's damage
+  # - Do nothing on unexpected cases
+  defp resolve_collisions(projectiles, players, obstacles, external_wall_id)
```

Comment:
Would be nice to rename this to `resolve_projectile_collisions` or something that explicitly says that its about projectiles

Reply:
Sure thing! nice suggestion